modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/renderer/ObjectEntryRowInfoItemRenderer.java

```diff
@@ -198,6 +202,28 @@ else if (Validator.isNotNull(
 
 						return StringPool.BLANK;
 					}
+					else if (Objects.equals(
```

Comment:
Could you reorder this condition? 
I believe this condition must be before this one:
`Objects.equals(objectField.getDBType(),ObjectFieldConstants.DB_TYPE_DATE)`